integration-cli/docker_cli_start_volume_driver_unix_test.go

```diff
@@ -82,9 +83,8 @@ func (s *DockerExternalVolumeSuite) SetUpSuite(c *check.C) {
 		s.ec.paths++
 
 		var pr pluginRequest
-		if err := json.NewDecoder(r.Body).Decode(&pr); err != nil {
-			http.Error(w, err.Error(), 500)
```

Comment:
Pretty sure we don't want to use `c.Assert` here either, same for the rest of the `http.Error` I guess. This part of the code creates a *mock*/*simple* volume plugin, shouldn't use `c.Assert` at all (the caller shall and will).